.github/workflows/release.yml

```diff
@@ -0,0 +1,68 @@
+name: Build RTSan Libs
+
+on:
+  push:
+    tags: ["v*"]
+
+
+jobs:
```

Comment:
Everything above this line is new, but below (until my next comment) is the exact same as the other already working build action